modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -876,23 +876,94 @@ public void testBindPublishedObjectDefinitions() throws Exception {
 
 		// Bind two published object definitions
 
+		ObjectDefinition objectDefinitionB =
+			_addAndPublishCustomObjectDefinition("B");
+		ObjectDefinition objectDefinitionBB =
+			_addAndPublishCustomObjectDefinition("BB");
```

Comment:
Why did you change from A and AA?

Reply:
Because I am using them in `AssertUtils.assertFailure` which requires final variables. So I couldn't use A and AA directly.